### PR TITLE
track de-re-activate

### PR DESCRIPTION
**Pull request by mkangia**
As a part of https://dimagi-dev.atlassian.net/browse/ICDS-1505 to track activate/deactivate

### CODE REVIEW

File: corehq/apps/users/views/mobile/users.py [hunk 1]
```diff
@@ -851,6 +852,8 @@ def _modify_user_status(request, domain, user_id, is_active):
         })
     user.is_active = is_active
     user.save(spawn_task=True)
+    log_model_change(request.user, user.get_django_user(), message={'is_active': is_active},
```
Comment (snopoke) [inline corehq/apps/users/views/mobile/users.py:855]:
```suggestion
    log_model_change(request.user, user.get_django_user(), fields_changed={'is_active': is_active},
```

**Review by snopoke — CHANGES_REQUESTED** [review]

File: corehq/apps/users/views/mobile/users.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/users/views/mobile/users.py:855]: @snopoke i thought of that too :) , but `fields_changed` is a list of fields that has changed. It does not have the value the fields were assigned,

Comment (snopoke) [reply corehq/apps/users/views/mobile/users.py:855]: does this even work? Looking at the code I don't see how message can be anything other than a string or a list

Comment (mkangia) [reply corehq/apps/users/views/mobile/users.py:855]: Yes @snopoke , this does work. `log_model_change` itself passes a [dict message](https://github.com/dimagi/commcare-hq/blob/1256dc7af10e9704532ccaf037d2bb5f9741edb8/corehq/util/model_log.py#L36).

Comment (snopoke) [reply corehq/apps/users/views/mobile/users.py:855]: I think that's a bug, it's saving to a charfield on the model. `LogEntry.objects.log_action` check if it's a list and serializes it to json if it is. How do these render on the UI as it is now?

Comment (mkangia) [reply corehq/apps/users/views/mobile/users.py:855]: an example attached in screenshot in https://github.com/dimagi/commcare-hq/pull/28058

Comment (snopoke) [reply corehq/apps/users/views/mobile/users.py:855]: this is how it should render: 
image

Comment (snopoke) [reply corehq/apps/users/views/mobile/users.py:855]:
```suggestion
    log_model_change(request.user, user.get_django_user(), message=f"Changed 'is_active' to {is_active}",
```

**Review by snopoke — APPROVED** [review]

**Pull request merged**